### PR TITLE
161918381 Add comparison data for trips and stops

### PR DESCRIPTION
**Pull request by rckopteri**
# Added
* transit-visualization page: Add date comparison summary data for trips and stops

### CODE REVIEW

File: ote/src/cljs/ote/views/transit_visualization/change_icons.cljs [hunk 1]
```diff
@@ -106,39 +106,44 @@
       [icon-l/icon-labeled
        [ic/content-remove-circle-outline {:color colors/icon-disabled}] nil]])])
 
-(defn change-icons
+(defn- show-added-trips [diff with-labels?]
+  [:div {:style {:flex "1"}}
+   [icon-l/icon-labeled
+    [ote-icons/outline-add-box {:color (if (= 0 (:added-trips diff))
```
Comment (philosotractor) [inline ote/src/cljs/ote/views/transit_visualization/change_icons.cljs]: Alternatively
```suggestion
    [ote-icons/outline-add-box {:color (if (zero? (:added-trips diff))
```

File: ote/src/cljs/ote/views/transit_visualization.cljs [hunk 2]
```diff
@@ -402,120 +402,132 @@
...
+                        (and (nil? departure-time-date1) departure-time-date2)
+                        "Uusi pysäkki reitillä"
+
+                        (not= departure-time-date1 departure-time-date2)
+                        [icon-l/icon-labeled [ic/action-query-builder]
+                         (time/format-minutes-elapsed
+                           (time/minutes-elapsed departure-time-date1 departure-time-date2))]
+
+                        :default
+                        [icon-l/icon-labeled {:style {:color "lightgray"}}
```
Comment (philosotractor) [inline ote/src/cljs/ote/views/transit_visualization.cljs]: Ok, but a good practise would be to define a logical symbol for a style/color and then define it in `colors.cljs`.
That way color/theme changes in future can be done simply by versioning/editing `colors.cljs` without changing any other source code for views.

File: ote/src/cljs/ote/views/transit_visualization/change_icons.cljs (hunk 1 above)
Comment (rckopteri) [reply ote/src/cljs/ote/views/transit_visualization/change_icons.cljs]: Thanks. Changed

File: ote/src/cljs/ote/views/transit_visualization.cljs (hunk 2 above)
Comment (rckopteri) [reply ote/src/cljs/ote/views/transit_visualization.cljs]: Yes. My bad. Changed now

**Pull request merged**